include/btu/bsa/settings.hpp

```diff
@@ -116,6 +116,7 @@ struct Settings
     std::vector<std::u8string> plugin_extensions;
     std::optional<std::vector<std::byte>> dummy_plugin;
     std::u8string dummy_extension;
+    bool plugin_bsa_limit;

```

Comment:
Rename to `dummy_plugin_has_loading_limit`. Or `dummy_plugin_loading_mode` and make it an enum

Reply:
Renamed, used enum for it.